src/Sentry.Unity/Default/UnknownPlatform.cs

```diff
@@ -11,14 +11,23 @@ public static class SentryUnknownPlatform
     {
         public static void Configure(SentryUnityOptions options)
         {
+            options.DiagnosticLogger?.Log(SentryLevel.Debug,
```

Comment:
```suggestion
            options.DiagnosticLogger?.Log(SentryLevel.Info,
```
What do you think about raising this to `Info`?